scipy/linalg/_special_matrices.py

```diff
@@ -55,6 +61,10 @@ def tri(N, M=None, k=0, dtype=None):
            [1, 1, 0, 0, 0]])
 
     """
+    warnings.warn("'tri'/'tril/'triu' is deprecated as of SciPy 1.11.0 in favour of"
```

Comment:
I would change the sentence a bit to avoid confusion, namely; 

```suggestion
    warnings.warn("'tri'/'tril/'triu' are deprecated as of SciPy 1.11.0 and will "
                   "be removed in v1.13.0. Please use numpy.(tri/tril/triu) instead."